apps/vessel-history/features/risk-summary/risk-summary.tsx

```diff
@@ -420,12 +443,17 @@ export function RiskSummary(props: RiskSummaryProps) {
         !hasVesselFlagsOnMOU ||
         !hasIUUIndicators ||
         !hasGapsIntentionalDisabling) && (
-        <RiskSection severity="none" title={t('risk.noRiskDetected', 'No risk detected') as string}>
+        <RiskSection
+          severity="none"
+          title={t('risk.noRiskDetected', 'No risk detected') as string}
+          loading={false}
```

Comment:
Why is this loading set to false always?

Reply:
because we are using the RiskSection component just as a div with a title and space, the spinners are in the components inside